PlayCover/Model/PlayApp.swift

```diff
@@ -138,6 +155,12 @@ class PlayApp: BaseApp {
         Uninstaller.clearExternalCache(info.bundleIdentifier)
     }
 
+    func removeAlias() {
+        if FileManager.default.fileExists(atPath: aliasURL.path) {
```

Comment:
`FileManager.default.delete` already checks if the file exists, so this would be a redundant check.